src/main/java/com/google/sps/servlets/ForumServlet.java

```diff
@@ -93,9 +93,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
 
     if (action.equals(ForumElement.ACCEPTED_PROPERTY)) {
```

Comment:
Just noticed now, but these should be "else if".

Reply:
Updated all in that section to else if